src/dmclock/test/CMakeLists.txt

```diff
@@ -1,3 +1,8 @@
+INCLUDE (CheckIncludeFiles)
```

Comment:
submodules have their own set of issues and it was determined that a subtree would be more simple. The problem is that it's not obvious in the source tree that dmclock is integrated in this way. github has a check for modifications to submodules; I wonder if we can do that with subtrees too. Alternatively there should be a README in src/dmclock explaining the issue.

Thanks for accommodating!